alphastats/gui/gui.py

```diff
@@ -18,6 +18,8 @@ def run():
     os.system(
         f"python -m streamlit run AlphaPeptStats.py --global.developmentMode=false {extra_args}"
     )
+
+    # TODO why are we starting the app a second time here?
```

Comment:
good question! weird thing, I checked and the functionality did not seem to chage